projects/shui/index.js

```diff
@@ -1,7 +1,11 @@
 const ADDRESSES = require('../helper/coreAssets.json')
 const SCFX_TokenAddress = "0x1858a8d367e69cd9E23d0Da4169885a47F05f1bE";
+const { sumTokensExport, } = require("../helper/unknownTokens");
+const MasterchefV2 = "0xeced26633b5c2d7124b5eae794c9c32a8b8e7df2";
```

Comment:
is this masterchef deployed/maintained by your team?

Reply:
No, it's made by nucleon I guess, we contact them,they said we could use it to calculate

Reply:
sorry, we cant count something as pool2 if the contract is not owned by your team.

Reply:
OK, that's fine. Do I need to lanuch a new PR or just delete the line?

Reply:
there is no change in the adapter except for that right? what info do you want to update here? https://defillama.com/protocol/shui#information